content/docs/api-reference/blockchain-api/index.md

```diff
@@ -25,15 +25,54 @@ Blockchain transactions can be fetched from the blockchain endpoint using the as
 assets/add30235-1424-4fda-840a-d5ef82c4c96f/events/11bf5b37-e0b8-42e0-8dcf-dc8c4aefc000
 ```
 
-To fetch all transactions for an asset event GET the blockchain resource:
+To fetch all transactions for an asset's events GET the blockchain resource:
 
 ```bash
 curl -v -X GET \
      -H "@$BEARER_TOKEN_FILE" \
      https://app.rkvst.io/archivist/v1alpha2/blockchain/assets/add30235-1424-4fda-840a-d5ef82c4c96f/events/11bf5b3
 ```
 
-Each of these calls returns a list of matching blockchain transactions in the form:
+Depending on the type of proof mechanism used, the response will be:
+
```

Comment:
it would be good to link to the advanced proof mechanism docs here, as someone who lands here may not understand what simple hash or khipu is and how they work, and therefore cannot easily parse the specific details in the response

Reply:
Agreed, link to proof mechanism information in advanced concepts section added.